projects/packages/forms/src/blocks/contact-form/util/merge-settings.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Merge the settings (or metadata) of two blocks. The settings of the second parameter
+ * take precedence over the first parameter. In the case of the `supports` and a`ttributes`
```

Comment:
Tiny comment typo :) 
```suggestion
 * take precedence over the first parameter. In the case of the `supports` and `attributes`
```